fake-manage-recalls-api/stubs/__files/update-recall.json

```diff
@@ -0,0 +1,5 @@
+{
+  "recallId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+  "nomsNumber": "A1234AA",
+  "recallLength": "14"
```

Comment:
Maybe ubiquitous language supports this being "recallLength" rather than "recallDuration" ... :-)
But am I right that in the TS it is "recallType"??